Tests/Application/.env

```diff
@@ -0,0 +1,4 @@
+APP_ENV=test
+DATABASE_URL=mysql://root:@127.0.0.1:3306/su_redirect_test
+DATABASE_CHARSET=utf8mb4
+DATABASE_COLLATE=utf8mb4_unicode_ci
```

Comment:
would like to add a `.editorconfig` file to automatically add a final newline 🙂 
https://github.com/sulu/sulu/blob/2.x/.editorconfig